### DISCUSSION TITLE
INVIO NOTIFICHE PROMEMORIA Data del promemoria (giorni)

### DISCUSSION THREAD

**Discussion by rlovisetto** [globaleaks/globaleaks-whistleblowing-software#3569] [General]
Abbiamo notato che il sistema invia 2 notifiche al giorno riferito al parametro "Data del promemoria (giorni)" configurato nel Canale,
Qualcuno mi sa dire se è configurabile la frequenza di invio delle notifiche?
Grazie
Roberto

**Comment by evilaliv3** [comment]
Il sistema invia una email al giorno per ogni giorno in cui è attiva una o
più email con promemoria attivo.

Alla pari di una sveglia il sistema continua a mandare email in ogni giorno
successivo finché l'allarme non viene disattivato

**Reply by rlovisetto** [reply]
Grazie Giovanni per la pronta risposta. Mi potresti far sapere anche quali sono le condizioni che fanno scattare la notifica? Questo tipo di promemoria non si accavalla con la notifica "Promemoria circa segnalazioni non lette o aggiornate"?

**Comment by evilaliv3** [comment]
Al momento i meccanismi di notifica sono per ora tutti implementati con semplicità e tutti indipendenti e hanno ragioni diverse.

-  La notifica di mancata risposta scatta dopo 7 giorni dall'ultima mancata lettura; ad esempio se oggi arriva una segnalazione o viene aggiornata una notifica verrà notificato fra 7 giorni  fino a che non avrà letto il contenuto inviato.
- il promemoria impostabile dall'utente scatta come una sveglia nella giornata in cui viene impostato e può successivamente essere disabilitato dagli utenti quando non ne hanno più bisogno

**Reply by rlovisetto** [reply]
Nella risposta si fa riferimento al promemoria impostabile dal Ricevente. La notifica alla quale mi riferisco io è quella che viene inviata dal sistema con frequenza giornaliera sino a quando il ricevente non legge la segnalazione.
Questo è un esempio dell'email che viene inviata:
"Gentile X Y

Questa email ti ricorda della presenza di segnalazioni non lette o aggiornate.
https://whistleblowing.nomesocietà.it/

Distinti saluti,
Whistleblowing - NOME SOCIETA'"

Non mi è chiaro quindi la differenza tra le due notifiche in quanto mi sembra che entrambe scattino sino a quando il ricevente non ha letto la segnalazione. C'è qualche cosa che mi sfugge?

**Reply by rlovisetto** [reply]
In aggiunta a quanto sopra riportato. Ho rilevato il seguente comportamento che mi sembrerebbe anomalo:
a) Ho configurato, nel canale di default, a "0" la "Data del premomeoria (giorni)" pensando che in questo modo venisse disabilitata la funzione
b) il giorno seguente ho rilevato che le notifiche sono state inviate nonostante avessi disabilitato la funzione. L'email che viene inviata presenta il seguente testo:
"Questa email è per notificarti che la data di promemoria di una o più segnalazioni è stata raggiunta."

**Comment by evilaliv3** [comment]
L'impostazione applicata sulla configurazione del canale determina la prima
configurazione di una segnalazione.

Avendola variata a 0 tutte le nuove segnalazioni vedranno disabilitato il
promemoria ma per tutte quelle precedentemente ricevute il promemoria
risulterà comunque sempre attivo (se attivo)
